### PR TITLE
Updates Serverless CLI v0.9.0 commands output

### PR DESCRIPTION
**Pull request by navidshaikh**
Labels:
 - branch/enterprise-4.1
 - branch/enterprise-4.2

### CODE REVIEW

**Comment by mgencur** [comment]
Looks good to me. QE Approved.

QE Approved

File: modules/serverless-workflow-basic.adoc [hunk 1]
```diff
@@ -55,40 +63,33 @@ The service's environment variable `TARGET` is now set to `Kn`.
...
 
 . Delete the service.
 +
 ----
 $ kn service delete hello
-
 Service 'hello' successfully deleted in namespace 'default'.
 ----
 +
 You can then verify that the 'hello' service is deleted by attempting to `list` it.
```
Comment (adellape) [inline modules/serverless-workflow-basic.adoc]: Consider marking up `hello`?

**Review by adellape — APPROVED** [review]
:+1:

File: modules/serverless-workflow-basic.adoc (hunk 1 above)
Comment (navidshaikh) [reply modules/serverless-workflow-basic.adoc]: Thanks for the review @adellape! updated the PR.

**Pull request merged**